examples/error_handling/README.md

```diff
@@ -0,0 +1,57 @@
+# Example - Error handling
+
+This guide demonstrates handling `DaprGrpcError` errors when using the Dapr python-SDK. It's important to note that not all Dapr gRPC status errors are currently captured and transformed into a `DaprGrpcError` by the SDK. Efforts are ongoing to enhance this aspect, and contributions are welcome. For detailed information on error handling in Dapr, refer to the [official documentation](https://docs.dapr.io/reference/).
```

Comment:
Its a nit, but I think we want Python capitalized.
```suggestion
This guide demonstrates handling `DaprGrpcError` errors when using the Dapr Python-SDK. It's important to note that not all Dapr gRPC status errors are currently captured and transformed into a `DaprGrpcError` by the SDK. Efforts are ongoing to enhance this aspect, and contributions are welcome. For detailed information on error handling in Dapr, refer to the [official documentation](https://docs.dapr.io/reference/).
```

Reply:
I don't want to retrigger some of the CI tasks because of this trivial docs change. Let's do that in a follow up PR after everything is merged here.